### ISSUE TITLE
getProfile throws error when run in Node 

### ISSUE THREAD

**Issue by zachferland** [uport-project/3box-js#176]
**Describe the bug**
Throws uncaught error, than finally throws once it tries to parse expected JSON, but finds error. Occurs when calling getProfile. 

**To Reproduce**
Run lib in node (occurred in multiple versions of node)

**Comment by zachferland** [comment]
May be related to general issues being seen around getprofile right now, and not anything specific to node. Will have to confirm once other issues is resolved.

**Comment by zachferland** [comment]
Checked again, same error still node, there is some error thrown and not caught in async function.

**Comment by oed** [comment]
What error is that? I definately had an error that was solved with #180

**Comment by zachferland** [comment]
@oed oh didn't see that, let me try again to confirm

**Comment by zachferland** [comment]
@oed all good! nice, closing
